src/tag.c

```diff
@@ -1476,6 +1476,16 @@ find_tags(
      * When the tag file is case-fold sorted, it is either one or the other.
      * Only ignore case when TAG_NOIC not used or 'ignorecase' set.
      */
+#ifdef FEAT_MULTI_LANG
+    /* Set a flag if the file extension is .txt */
+    int is_txt = 0;
```

Comment:
there is a statment before this declaration. vim doesn't support C99. so move up this declaration with #ifdef FEAT_MULTI_LANG.